src/clients/authorization/Settings.tsx

```diff
@@ -0,0 +1,174 @@
+import React, { useState } from "react";
+import { useTranslation } from "react-i18next";
+import { Controller, useForm } from "react-hook-form";
+import {
+  Button,
+  Divider,
+  FormGroup,
+  PageSection,
+  Radio,
+  Spinner,
+  Switch,
+} from "@patternfly/react-core";
+
+import type ResourceServerRepresentation from "@keycloak/keycloak-admin-client/lib/defs/resourceServerRepresentation";
+import { useAdminClient, useFetch } from "../../context/auth/AdminClient";
+import { FormAccess } from "../../components/form-access/FormAccess";
+import { HelpItem } from "../../components/help-enabler/HelpItem";
+import { SaveReset } from "../advanced/SaveReset";
+
+const PolicyEnforcementModes = ["ENFORCING", "PERMISSIVE", "DISABLED"] as const;
+const DecisionStrategy = ["UNANIMOUS", "AFFIRMATIVE"] as const;
```

Comment:
Can we use constant casing for these?
```suggestion
const POLICY_ENFORCEMENT_MODES = ["ENFORCING", "PERMISSIVE", "DISABLED"] as const;
const DECISION_STRATEGY = ["UNANIMOUS", "AFFIRMATIVE"] as const;
```